src/Identifier/LdapIdentifier.php

```diff
@@ -92,17 +92,17 @@ public function __construct(array $config = [])
      */
     protected function _checkLdapConfig()
     {
-        if (!isset($this->_config['bindDN'])) {
-            throw new RuntimeException('Config `bindDN` is not set.');
+        if ($this->getConfig('bindDN') === null) {
+            throw new RuntimeException('Config key `bindDN` is not set.');
         }
-        if (!is_callable($this->_config['bindDN'])) {
+        if (!is_callable($this->getConfig('bindDN'))) {
             throw new InvalidArgumentException(sprintf(
-                'The `bindDN` config is not a callable. Got `%s` instead.',
-                gettype($this->_config['bindDN'])
+                'The value of `bindDN` config key is not a callable. Got `%s` instead.',
+                gettype($this->getConfig('bindDN'))
             ));
         }
-        if (!isset($this->_config['host'])) {
-            throw new RuntimeException('Config `host` is not set.');
+        if ($this->getConfig('host') === null || trim($this->getConfig('host') === '')) {
```

Comment:
Remove the `trim()`, if someone sets invalid host name with whitespace let an error be generated.